manifest.json

```diff
@@ -8,13 +8,19 @@
     },
     "version": "1.0~ynh5",
     "url": "https://github.com/hgentry/81/",
+    "upstream": {
+        "license": "MIT",
+        "website": "https://github.com/hgentry/81/",
+        "demo": "https://hgentry.github.io/81/",
+        "code": "https://github.com/hgentry/81"
+    },
     "license": "MIT",
     "maintainer": {
         "name": "eric_G",
         "email": ""
     },
     "requirements": {
-        "yunohost": ">= 4.1.7"
+        "yunohost": ">= 4.2.4"
```

Comment:
This may be the cause of the 403 issue :wink: 

The behavior of `ynh_setup_source` changes if the manifest requires a version higher than 4.2, and in that case applies some new default permissions: https://github.com/YunoHost/yunohost/blob/9bec81398ab26cbe04516052308a8901ca697363/data/helpers.d/utils#L771-L779

(Also related is that damn fact that `useradd` won't create user `243` because it thinks it's a uid :/ )